### PR TITLE
Fix incorrect comment in Kafka.InternalField

### LINKED ISSUE #5752: Fix incorrect comment in KafkaInternalFieldManager.InternalField
```java
        ColumnMetadata getColumnMetadata(boolean hidden)
        {
            return ColumnMetadata.builder()
                    .setName(columnName)
                    .setType(type)
                    .setComment(Optional.ofNullable(columnName))
                    .setHidden(hidden)
                    .build();
        }
```
`.setComment(Optional.ofNullable(columnName))`
should be 
`.setComment(Optional.ofNullable(comment))`

### PR DESCRIPTION
**Pull request by brandboat**
Fixes #5752

### CODE REVIEW

File: presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaInternalFieldManager.java [hunk 1]
```diff
@@ -125,7 +125,7 @@ ColumnMetadata getColumnMetadata(boolean hidden)
             return ColumnMetadata.builder()
                     .setName(columnName)
                     .setType(type)
-                    .setComment(Optional.ofNullable(columnName))
+                    .setComment(Optional.ofNullable(comment))
```
Comment (kokosing) [inline presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaInternalFieldManager.java:128]: can we have an unit test for that?

Comment (brandboat) [reply presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaInternalFieldManager.java:128]: No problem.

**Comment by brandboat** [comment]
Is there anything else need to be revised ?

**Review by kokosing — APPROVED** [review]

**Comment by kokosing** [comment]
Merged, thanks!

**Pull request merged**